### ISSUE TITLE
I want to set RNN type in initializing SequenceTagger 

### ISSUE THREAD

**Issue by hayata-yamamoto** [zalandoresearch/flair#1100]
To be removed, once it is done: Please add the appropriate label to this ticket, e.g. feature or enhancement.

**Is your feature/enhancement request related to a problem? Please describe.**

I want to change SequenceTagger in initializing SequenceTagger. 

**Describe the solution you'd like**
Please provide suggestions on how to solve the problem, how to implement the idea.

* add default setting of instance variables on __init__ method

for example, 

```python

class SequenceTagger(flair.nn.Model):
    def __init__(
        self,
        hidden_size: int,
        embeddings: TokenEmbeddings,
        tag_dictionary: Dictionary,
        tag_type: str,
        use_crf: bool = True,
        use_rnn: bool = True,
        rnn_layers: int = 1,
        dropout: float = 0.0,
        word_dropout: float = 0.05,
        locked_dropout: float = 0.5,
        train_initial_hidden_state: bool = False,
        pickle_module: str = "pickle",
        relearn_embeddings: bool = True,
        bidirectional: bool = True, 
        rnn_type: str = 'LSTM'
    ):
        """
        Initializes a SequenceTagger
        :param hidden_size: number of hidden states in RNN
        :param embeddings: word embeddings used in tagger
        :param tag_dictionary: dictionary of tags you want to predict
        :param tag_type: string identifier for tag type
        :param use_crf: if True use CRF decoder, else project directly to tag space
        :param use_rnn: if True use RNN layer, otherwise use word embeddings directly
        :param rnn_layers: number of RNN layers
        :param dropout: dropout probability
        :param word_dropout: word dropout probability
        :param locked_dropout: locked dropout probability
        :param train_initial_hidden_state: if True, trains initial hidden state of RNN
        """

        super(SequenceTagger, self).__init__()
        if rnn_type not in ['LSTM', 'GRU']: 
            raise ValueError('RNN type must be LSTM or GRU.')
```

**Additional context**
Add any other context or screenshots about the feature/enhancement request.

**Comment by alanakbik** [comment]
Hello @hayata-yamamoto we just added this option to master branch. Can you check it out to see if it works for you?

**Comment by hayata-yamamoto** [comment]
@alanakbik 
Excellent!! At a glance, it will work well. if i have some problem, let you know by new issue. 
Thanks a lot!
